src/validators/module/index.js

```diff
@@ -1,8 +1,9 @@
 import {noop} from 'lodash'
+import validateLoadersEntry from './validateLoadersEntry'
+
 export default [
-  {key: 'module.loaders', validate: noop},
-  {key: 'module.preLoaders', validate: noop},
-  {key: 'module.postLoaders', validate: noop},
+  {key: 'module.loaders', validate: validateLoadersEntry},
```

Comment:
`Entry` doesn't make sense in this context. `validateLoaders` should do just fine